sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3555,6 +3564,9 @@ class SQLConf extends Serializable with Logging {
 
   def coalesceShufflePartitionsEnabled: Boolean = getConf(COALESCE_PARTITIONS_ENABLED)
 
+  def adaptiveCustomCostEvaluatorClass: Option[String] =
```

Comment:
nit: we don't have to create a method here if it's only called once

Reply:
@cloud-fan - sure, updated.